src/Resources/assets/js/contao-tinymce-bundle.js

```diff
@@ -29,16 +33,142 @@ class TinyMceBundle {
             }
 
             utilsBundle.util.runRecursiveFunction(importPlugin, ['paste', 'link', 'lists'], () => {
-                config.selector = '#' + element.id;
-
-                // set language
-                config.language_url = (typeof NON_ENCORE !== 'undefined' && NON_ENCORE ? '/bundles/heimrichhannottinymce/js/languages/' : '/build/tinymce/languages/') + language + '.js';
-                config.language = language;
+                // modify config before initialization
+                config = TinyMceBundle.modifyConfig(element, config);
 
                 // init
                 tinymce.init(config);
+
+            });
+
+            let form = element.closest('form');
+            if(!form.classList.contains('tinymce-form')) {
+                form.classList.add('tinymce-form');
+            }
+        });
+
+    }
+
+    static initEventListeners() {
+        utilsBundle.event.addDynamicEventListener('click', '.tinymce-limitChars', (element) => {
+            let message = element.closest('.tinymce-limit-error'),
+                editorTarget = message.dataset.tinymce,
+                limit = message.dataset.limit,
+                editor = tinymce.get(editorTarget);
+
+            if(!editor || !limit) {
+                return;
+            }
+
+            editor.getBody().textContent = editor.getBody().textContent.substring(0, limit);
+            message.remove();
+            TinyMceBundle.resetTinymceError(editor);
+        });
+
+        utilsBundle.event.addDynamicEventListener('submit', '.tinymce-form', (form, e) => {
+            let errors = form.querySelectorAll('.tinymce-error');
+
+            if(!errors) {
+                return;
+            }
+
+            e.preventDefault();
+            errors[0].scrollIntoView();
+        });
+    }
+
+
+    static modifyConfig(element, config) {
+        config.selector = '#' + element.id;
+
+        // set language
+        let language = document.querySelector('html').getAttribute('lang');
+        config.language_url = (typeof NON_ENCORE !== 'undefined' && NON_ENCORE ? '/bundles/heimrichhannottinymce/js/languages/' : '/build/tinymce/languages/') + language + '.js';
+        config.language = language;
+
+        // set eventlisteners for tinymce editor
+        config.setup = (editor) => {
+            TinyMceBundle.getTinyMceSetup(editor, config);
+        };
+
+        return config;
+    }
+
+
+    static getTinyMceSetup(editor, config) {
+        // set maxChars event listener
+        if('undefined' !== config.maxChars) {
```

Comment:
This needs to be:  if( typeof 'undefined' !== config.maxChars)

Reply:
shouldn't it be if( 'undefined' !== typeof config.maxChars) ?